10.9-libcxx/stable/main/finkinfo/libs/perlmods/module-build-pm-14.0.info

```diff
@@ -3,7 +3,7 @@ Package: module-build-pm
 Version: 0.42.34
 Revision: 1201
 Epoch: 1
-Distribution: 14.0
+Distribution: 14.0, 15.0
```

Comment:
This triggers a validator error because the Distribution value here does not match the value in the .info filename itself. Hanspeter pushed a module-build-pm-15.0.info recently that handles that distro.